vyxal/transpile.py

```diff
@@ -90,9 +90,9 @@ def transpile_token(
         # screws up escape sequences.
 
         # So instead, we have to manually escape the string
-        string = string.replace("\\", "\\\\").replace('"', '\\"')
-        string = string.replace("\n", "\\n")
-        return indent_str(f'stack.append("{string}")', indent)
+        string = f"{string!r}"
+        string = re.sub(r"\\([ntbrf])", r"\1", string).replace("\\`", "`")
```

Comment:
I'm pretty sure there should be a better way of doing this, but meh. Also, I think you should write some unit tests for this particular bit of code, because......